packages/component/src/Styles/defaultStyleOptions.js

```diff
@@ -5,14 +5,13 @@ function fontFamily(fonts) {
 }
 
 const DEFAULT_ACCENT = '#0063B1';
-const DEFAULT_SUBTLE = '#767676'; // With contrast 4.5:1 to white
 const PADDING_REGULAR = 10;
+const DEFAULT_SUBTLE = '#767676'; // With contrast 4.5:1 to white
```

Comment:
Why did this like get moved? Should be in alphabetical order

Reply:
I'll move it back. Didn't look close enough. Thought it said 'DEFAULT_REGULAR'.